api/v1alpha1/wildflyserver_types.go

```diff
@@ -67,6 +67,40 @@ type WildFlyServerSpec struct {
 	// ResourcesSpec defines the resources used by the WildFlyServer, ie CPU and memory, use limits and requests.
 	// More info: https://pkg.go.dev/k8s.io/api@v0.18.14/core/v1#ResourceRequirements
 	Resources *corev1.ResourceRequirements `json:"resources,omitempty"`
+	// LivenessProbe defines the periodic probe of container liveness. Container will be restarted if the probe fails
+	LivenessProbe *ProbeSpec `json:"livenessProbe,omitempty"`
+	// ReadinessProbe defines the periodic probe of container service readiness. Container will be removed from service endpoints if the probe fails.
+	ReadinessProbe *ProbeSpec `json:"readinessProbe,omitempty"`
+	// StartupProbe indicates that the Pod has successfully initialized. If specified, no other probes are executed until this completes successfully.
+	// If this probe fails, the Pod will be restarted, just as if the livenessProbe failed. This can be used to provide different probe parameters at the beginning of a Pod's lifecycle,
+	// when it might take a long time to load data or warm a cache, than during steady-state operation.
+	StartupProbe *ProbeSpec `json:"startupProbe,omitempty"`
+}
+
+// ProbeSpec Probe describes a health check to be performed against a container to determine whether it is alive or ready to receive traffic.
+// +k8s:openapi-gen=true
+type ProbeSpec struct {
+	// Number of seconds after the container has started before liveness probes are initiated.
+	// Defaults to 60 seconds. Minimum value is 0.
+	// +optional
+	InitialDelaySeconds int32 `json:"initialDelaySeconds,omitempty"`
+	// Number of seconds after which the probe times out.
+	// Defaults to 1 second. Minimum value is 1.
+	// More info: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle#container-probes
+	// +optional
+	TimeoutSeconds int32 `json:"timeoutSeconds,omitempty"`
+	// How often (in seconds) to perform the probe.
+	// Default to 10 seconds. Minimum value is 1.
+	// +optional
+	PeriodSeconds int32 `json:"periodSeconds,omitempty"`
+	// Minimum consecutive successes for the probe to be considered successful after having failed.
+	// Defaults to 1. Must be 1 for liveness and startup. Minimum value is 1.
+	// +optional
+	SuccessThreshold int32 `json:"successThreshold,omitempty"`
```

Comment:
`// +kubebuilder:validation:Minimum=1`

Reply:
I've added the missing` // +kubebuilder:validation:Minimum=1`

Reply:
missed this one, ` // +kubebuilder:validation:Minimum=1` is still missing for `FailureThreshold`

Reply:
Correct, fixed. I've just added // +kubebuilder:validation:Minimum=1 under `FailureThreshold`